src/app.ts

```diff
@@ -3994,6 +3994,72 @@ class App {
       }
     });
 
+    router.get("/accounting-activity", async (req: Request, res: Response) => {
+      try {
+        const getAccountingActivityAccountUsage = await xero.financeApi.getAccountingActivityAccountUsage(req.session.activeTenant.tenantId);
+        const getAccountingActivityLockHistory = await xero.financeApi.getAccountingActivityLockHistory(req.session.activeTenant.tenantId);
+        const getAccountingActivityReportHistory = await xero.financeApi.getAccountingActivityReportHistory(req.session.activeTenant.tenantId);
+        const getAccountingActivityUserActivities = await xero.financeApi.getAccountingActivityUserActivities(req.session.activeTenant.tenantId);
+
+        res.render("accounting-activity", {
+          consentUrl: await xero.buildConsentUrl(),
+          authenticated: this.authenticationData(req, res),
+          getAccountingActivityAccountUsage: getAccountingActivityAccountUsage.body,
+          getAccountingActivityLockHistory: getAccountingActivityLockHistory.body,
+          getAccountingActivityReportHistory: getAccountingActivityReportHistory.body,
+          getAccountingActivityUserActivities: getAccountingActivityUserActivities.body
+        });
+      } catch (e) {
+        res.status(res.statusCode);
+        res.render("shared/error", {
+          consentUrl: await xero.buildConsentUrl(),
+          error: e
+        });
+      }
+    });  
+
+    router.get("/cash-validation", async (req: Request, res: Response) => {
+      try {
+        const getCashValidation = await xero.financeApi.getCashValidation(req.session.activeTenant.tenantId);
+
+        res.render("cash-validation", {
+          consentUrl: await xero.buildConsentUrl(),
+          authenticated: this.authenticationData(req, res),
+          getCashValidation: getCashValidation.body
+        });
+      } catch (e) {
+        res.status(res.statusCode);
+        res.render("shared/error", {
+          consentUrl: await xero.buildConsentUrl(),
+          error: e
+        });
+      }
+    });  
+
+    router.get("/financial-statement", async (req: Request, res: Response) => {
+      try {
+        const getFinancialStatementBalanceSheet = await xero.financeApi.getFinancialStatementBalanceSheet(req.session.activeTenant.tenantId);
+        const getFinancialStatementCashflow = await xero.financeApi.getFinancialStatementCashflow(req.session.activeTenant.tenantId);
+        const getFinancialStatementProfitAndLoss = await xero.financeApi.getFinancialStatementProfitAndLoss(req.session.activeTenant.tenantId);
+        const getFinancialStatementTrialBalance = await xero.financeApi.getFinancialStatementTrialBalance(req.session.activeTenant.tenantId);
+
+        res.render("financial-statement", {
+          consentUrl: await xero.buildConsentUrl(),
```

Comment:
This isn't related to your specific changes, I can see you're simply following the pattern already utilised.

But this local is set in every render call.  To keep it DRY, this could be set via middleware.